src/components/alert/alert.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { describe, expect, it } from "vitest";
+import { getByText, render } from "@testing-library/react";
+import React from "react";
+import { Alert } from "./alert";
+
+describe("Alert", () => {
+    it("renders an alert with a title and children", () => {
+        // ARRANGE
+        const { getByRole } = render(
```

Comment:
Using `screen.getByRole` is recommended by the creators, although it doesn't make a substantial difference 😄

Reply:
I will change it to match the docs! Thanks for bringing that up. I run through different examples to get the additional "toBeInTheDocument()" functions to work and I guess I messed it up.